### PR TITLE
Small addenda to BQ doc: expenses, billing project, and sync tables

### PR DESCRIPTION
**Pull request by rafatower**
Some doc about how expenses work. Two important things:
- [x] the oauth billing project is to be tested and confirmed
- [x] I forgot to add some advise about setting https://cloud.google.com/bigquery/docs/custom-quotas

(bottom line: let's leverage google quotas)

### CODE REVIEW

File: doc/developer-center/import-api/guides/07-database-connectors.md [hunk 1]
```diff
@@ -1298,3 +1298,23 @@ from_external_source | Has the value **false** for all connector-based synchroni
...
+
+#### BigQuery pricing and expenses
+
+This service is subject to charges in your BigQuery project, according to your pricing settings. Please check https://cloud.google.com/bigquery/pricing for more information.
+
+##### The billing project
+
+The project used to bill expenses is either:
+- the billing project associated to your user if you configured it through `Settings > Account > Connect to external data sources`
+- PLANNED the one set through the `billing_project` parameter, which is mandatory when using authorization through Service Account credentials.
```
Comment (manmorjim) [inline doc/developer-center/import-api/guides/07-database-connectors.md]: **Question** :question: : with _"is mandatory when using authorization through Service Account credentials"_ you mean OAuth2? AFAIK Service Account credentials have associated the billing project so in that case `billing_project` parameter should be ignored.

Comment (rafatower) [reply doc/developer-center/import-api/guides/07-database-connectors.md]: No, I don't. Let me clarify it:

- service account is that json file with secrets, not oauth2
- a service account is always associated with a project, which turns out to be the default and billing project.

From the docs:
https://cloud.google.com/bigquery/pricing#how_charges_are_billed
> Each project you create has a billing account attached to it. Any charges incurred by BigQuery jobs run in the project are billed to the attached billing account. BigQuery storage charges are also billed to the attached billing account.

In other words: `service_account_info.json -> billing_project -> charges`.

In the case of Oauth2, permissions to the user are granted explicitly, as shown by our tests.

Comment (manmorjim) [reply doc/developer-center/import-api/guides/07-database-connectors.md]: oohhh, I got it!

Comment (alonsogarciapablo) [reply doc/developer-center/import-api/guides/07-database-connectors.md]: `which is mandatory when using authorization through Service Account credentials.` --> might be confusing as users might think that Service Account authorization is available?

I guess we can removed `PLANNED` as it's already implemented. Thanks!

**Pull request merged**